frontend/src/App.jsx

```
@@ -1,35 +1,32 @@
+      <div className="absolute bottom-0 -right-4 size-96 bg-cyan-500 opacity-20 blur-[100px]" />
+      <button className="btn btn-red-300 z-10 cursor-pointer" onClick={login}>
+        Login
+      </button>
 
-export default App
+      <Routes>
+        <Route path="/" element={<Chatpage />}></Route>
+        <Route path="/signup" element={<SignUp />}></Route>
+        <Route path="/login" element={<LoginPage />}></Route>
+      </Routes>
+    // </div>
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)